apps/balancer-tools/src/app/apr/round/(components)/MultiSelectDropdown.tsx

```diff
@@ -0,0 +1,172 @@
+          setSelectedItems,
+          onSelectionItemsChange,
+        )}
+      >
+        {({
+          getLabelProps,
+          getInputProps,
+          getItemProps,
+          highlightedIndex,
+          isOpen,
+          toggleMenu,
+          inputValue,
+        }) => {
+          return (
+            <div>
+              <div className="relative mt-1">
+                <div className="flex items-center bg-blue6 relative w-full cursor-default overflow-hidden rounded-lg text-left shadow-md focus:outline-none focus-within:ring-2 focus-within:ring-blue6 focus-within:ring-opacity/75 focus-within:ring-offset-2 focus-within:ring-offset-blue3 sm:text-sm">
+                  <label {...getLabelProps()} className="flex-1 mx-1">
+                    <MagnifyingGlassIcon />
+                  </label>
+                  {selectedItems.map((value, idx) => {
+                    return (
+                      <button
+                        key={value + idx}
+                        className="relative mx-1 flex gap-2 h-fit items-center rounded-md px-2 py-1 bg-blue7 whitespace-nowrap"
+                        onClick={() =>
+                          removeSelectedItemByIndex(
+                            idx,
+                            selectedItems,
+                            setSelectedItems,
+                            onSelectionItemsChange,
+                          )
+                        }
+                      >
+                        <span>{value}</span>
+                        <span>
+                          <Cross1Icon />
+                        </span>
+                      </button>
+                    );
+                  })}
+
+                  <input
+                    placeholder={labelText}
+                    {...getInputProps()}
+                    onFocus={() => {
+                      toggleMenu();
+                    }}
+                    onKeyDown={(e) => {
+                      handleOKeyDown(e);
+                    }}
+                    className="w-full border-none py-2 pl-3 pr-10 text-sm leading-5 outline-none text-white bg-transparent"
+                    type="text"
+                  />
+                </div>
+              </div>
+
+              {isOpen && (
+                <div className="absolute z-50 my-2 flex max-h-52 flex-col overflow-y-scroll rounded border-[1px] border-blue6 bg-blue3 scrollbar-thin scrollbar-track-blue2 scrollbar-thumb-slate12 w-full">
+                  {items.filter(
+                    (item) =>
+                      !selectedItems.find((selected) => selected === item) &&
+                      item.includes(String(inputValue)),
+                  ).length > 0 ? (
+                    items
+                      .filter(
+                        (item) =>
+                          !selectedItems.find(
+                            (selected) => selected === item,
+                          ) && item.includes(String(inputValue)),
```

Comment:
could filter this in the component/assign to another variable to this repetition here